docs/tool/js/views/filter-view.js

```diff
@@ -978,7 +979,19 @@ var filterView = {
         }
         filterView.indicateActivatedFilters();
     },
-
+    clearLocationBasedFilters: function(){
+        function isLocationBased(filterControl){
+            return (
+                filterControl.component.hasOwnProperty('zones') || 
```

Comment:
The part of the configuration in dataChoices.js that determines if it's a zone-based data set is
`"data_level": "zone",`

It would be better to use that property  instead of whether it has a list of zones.